src/pds_doi_service/core/input/pds4_util.py

```
@@ -239,9 +512,13 @@ def process_pds4_fields(self, pds4_fields):
 
             editors = self.get_editor_names(pds4_fields["editors"].split(";")) if "editors" in pds4_fields else None
 
-            # The 'authors' field is inconsistent on the use of separators.
-            # Try to make a best guess on which method is better.
-            o_best_method = self._find_method_to_parse_authors(pds4_fields["authors"])
+            # Handle authors field - check if it exists before processing
+            # 20250723 -- process <author_list> as optional
+            authors_list = []
+            if "authors" in pds4_fields:
```

@rsjoyner aren't authors required in DOI metadata? should we raise an exception if we do not find authors?

---

You are correct.  That is legacy code.  I intentionally did not modify legacy code -- at least not at this point.
There is a fair amount of legacy code that I would probably change.  

You can identify my code with:  # 20250501...